deb-package-builder/functions.sh

```diff
@@ -47,7 +47,7 @@ download_from_pecl()
         with_retry "pecl download ${PECL_PACKAGE_NAME}" 6
         # determine the downloaded version
         EXT_VERSION=$(ls ${PACKAGE_SHORT_NAME}-*.tgz | \
-                sed "s/${PACKAGE_SHORT_NAME}-\([0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*\)\.tgz/\1/")
+                sed -E "s/${PACKAGE_SHORT_NAME}-([0-9]+.[0-9]+(.[0-9]+)+).tgz/\1/")
```

Comment:
Do we need to keep the escaping for the "." characters?

Reply:
Good catch, done